web/src/pages/api/user/register.ts

```diff
@@ -45,7 +47,7 @@ export default async function Register(req: Request, res: NextApiResponse) {
         },
       });
 
-    const confirmEmailAPIRoute = `http://localhost:3000/api/user/confirmEmail/${newUser.id}`;
+    const confirmEmailAPIRoute = `http://localhost:3000/api/user/confirmEmail/id?=${newUser.id}token?=${jwt}`;
```

Comment:
We should add a check here to assign the right hostname. Also the slug seems to be wrong. Here is what I believe is the correct code with the environment check:
```
  const url =
    process.env.NODE_ENV === "development"
      ? "http://localhost:3000"
      : "https://daydream.wtf";
  const confirmEmailAPIRoute = `${url}/api/user/confirmEmail?userId=${newUser.id}&token=${jwt}`;
```